data/outfits/ship-stats.txt

```diff
@@ -341,6 +431,13 @@ outfit "Ruin-The-Fun Stat Scan asteroid scan power"
 	unplunderable 1
 	"asteroid scan power" 1
 
+outfit "Ruin-The-Fun Stat Scan asteroid scan power * 100000000"
+	"display name" "asteroid scan power * 100000000"
```

Comment:
Some outfit names are not displayed entirely due to their display name being too long, hiding the attribute name:
![image](https://github.com/Pshy0/es-ruin-the-fun/assets/44928041/828a4fd4-55e2-4669-bd98-3e585b26c5e9)
![image](https://github.com/Pshy0/es-ruin-the-fun/assets/44928041/080d5a8d-6219-4180-a8bd-20f3662d9d69)

I suggest replacing ` * 100000000` by ` * 10^8`. Lower values do not need replacing as long as it fits the screen.

The existing tactical scan stat outfit also have this problem.